README.md

```diff
@@ -29,3 +29,9 @@ Create a Wordpress Plugin that would connect to it. 🚀
 - [VS CODE](https://code.visualstudio.com/)
 - [POSTMAN](https://www.postman.com/)
 - [WP CODEX](https://codex.wordpress.org/Main_Page)
+### Deck of Cards
+### Steps for use the plugin
+1) Locate the «Cards wigdet» plugin on Plugins > Installed Plugins and click on the «Activate» option
+2) Go to Edit Page > Widgets
```

Comment:
Second step is not clear, as I'm not able to find the Widget within any page edit after doing activation:

![image](https://user-images.githubusercontent.com/35984989/141382450-9f5c743f-6c50-4646-8fec-bbf4b2ea261b.png)

Reply:
I'm sorry, It was my mistake, please go to the homepage and  go to "Customize> Widgets" 
<img width="321" alt="error" src="https://user-images.githubusercontent.com/30477423/141386714-d9217a9a-8711-4755-a4bc-fb098a0e3dc0.PNG">
<img width="216" alt="error2" src="https://user-images.githubusercontent.com/30477423/141386876-cb08fc6e-b340-4bc3-81b0-71581730da59.PNG">
Depending on your theme, several areas will be shown where to place the widget, I recommend using the "zakra" theme and placing it as "sidebar right", as follows:
<img width="223" alt="sidebarRight" src="https://user-images.githubusercontent.com/30477423/141387166-09ba6dca-1837-4f3d-9517-e13262b07649.PNG">
<img width="164" alt="plus_option" src="https://user-images.githubusercontent.com/30477423/141388041-bfab07bd-bb86-4501-9720-ce3c9d1da82b.PNG">

<img width="197" alt="widget_selection" src="https://user-images.githubusercontent.com/30477423/141387867-70641b64-3180-4c5b-b872-05daaf784180.PNG">

It will look like these:
<img width="185" alt="widget" src="https://user-images.githubusercontent.com/30477423/141387203-d84162af-216b-4f63-b681-933a5c3a59b8.PNG">
And It will look like these on the client's side:
<img width="948" alt="final_result" src="https://user-images.githubusercontent.com/30477423/141387459-c72a6594-1b7d-4520-9004-2eb3ea76b94c.PNG">

Reply:
Thank you leone. 

But next time make sure to make an ammend commit as review comments are not a good place to keep relevant information visible.